python/docs/Makefile

```diff
@@ -7,7 +7,7 @@ SPHINXBUILD   = sphinx-build
 PAPER         =
 BUILDDIR      = _build
 
-export PYTHONPATH=$(realpath ..):$(realpath $(SPARK_HOME)/python)
+export PYTHONPATH=$(realpath ..):$(realpath $(SPARK_HOME))/python
```

Comment:
change this line to `export PYTHONPATH=$(realpath ..):$(realpath $(SPARK_HOME))/python:$(realpath $(SPARK_HOME))/python/lib/py4j-0.8.2.1-src.zip`